### ISSUE TITLE
RTX 5090 via Pinokio - Black Screen Output - RuntimeWarning: invalid value encountered in cast

### ISSUE THREAD

**Issue by Godskull17** [lllyasviel/FramePack#159]
The model runs without error after a clean install of FramePack via Pinokio on Windows. Still, the final video output is a black screen, and no useful preview or "next latents" are shown in the interface.

The CLI does not crash, and inference completes, but the preview rendering fails silently, resulting in either:

1. A blank .mp4
2. An empty or black preview window
3. No video output at all

Logs (partial):
preview = (preview * 255.0).detach().cpu().numpy().clip(0, 255).astype(np.uint8)
RuntimeWarning: invalid value encountered in cast
System:

OS: Windows 11
Installed via: Pinokio
GPU: ZOTAC NVIDIA RTX 5090 32GB

![Image](https://github.com/user-attachments/assets/a975079b-b1ce-43a0-a12b-c23e00e691e7)

**Comment by BahamutRU** [comment]
Blackwell work only with CUDA 12.8 — do you update CUDA and Torch?

I find this software few minutes ago, and machine with 5090 is offline at this moment, I can't check, but tomorrow…

**Comment by Godskull17** [comment]
Could you guide me?

**Comment by BahamutRU** [comment]
> Could you guide me?

Sry, I skipped this test. =( I hope, you fixed your problem.

**Comment by ziws-z** [comment]
Same problem here !
rtx4080 super 16go vram
